client/src/components/DataFiles/DataFiles.js

```diff
@@ -107,6 +107,7 @@ const DataFiles = () => {
   );
```

Comment:
On line [97](https://github.com/TACC/Core-Portal/pull/449/files#diff-f932679a560d01e39cb1e24ce8dc89557ab7c09864ac63187be124da3115eb5cR97) above, we have a `<PrivateDataRedirect />` [component](https://github.com/TACC/Core-Portal/blob/ecd751b66d094675ffc1c8e926bc12b6b64b143a/client/src/components/DataFiles/DataFiles.js#L30) to redirect from `/workbench/data` to the first system in the `systems` array, which we assume is the private `My Data` system.  Can we refactor that component to redirect to the first system in the list, and use the correct corresponding `scheme` and `api`?

Reply:
@rstijerina 👍🏼